File: CS50x/sentimental-mario-more/mario.py
from cs50 import get_int
height = 0

# Loop until you get a number between 1 and 8
while height > 8 or height < 1:
    height = get_int("Height: ")

# Now print the spaces, the hashes double space and the other side of the hashes
for i in range(height):
    print(" " * (height - (i + 1)), end=(""))
    print("#" * (i + 1), end=(""))
    print("  ", end=(""))
    print("#" * (i + 1))

File: CS50x/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("How high? ");
    } while (n < 1 || n > 8);

    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            if (i + j < n - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }

        printf("  ");

        for (int j = n - i; j <= n; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}